src/igvfd/audit/construct_library_set.py

```diff
@@ -139,7 +139,10 @@ def audit_construct_library_set_orf_gene(value, system):
         orf_ids = value.get('orf_list')
         for o in orf_ids:
             orf_object = system.get('request').embed(o + '@@object?skip_calculated=true')
-            if 'gene' in orf_object:
+            if 'genes' in orf_object:
+                for d in orf_object['genes']:
+                    orf_genes.add(d)
+            elif 'gene' in orf_object:
```

Comment:
remove this check, the property has been deprecated

Reply:
Removed